### PR TITLE
tecoc: 20150606 -> unstable-2020-11-03

### PR DESCRIPTION
**Pull request by AndersonTorres**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/editors/tecoc/default.nix [hunk 1]
```diff
@@ -1,37 +1,47 @@
...
-    cp src/aaout.txt doc/* $out/share/doc/${pname}-${version}
-    cp lib/* lib2/* $out/lib/teco/macros
+    runHook preInstall
+    install -m755 src/tecoc $out/bin
+    install -m644 src/aaout.txt doc/* $out/share/doc/${pname}-${version}
+    install -m644 lib/* lib2/* $out/lib/teco/macros
+    runHook postInstall
+  '';
+
+  postInstall = ''
```
Comment (SuperSandro2000) [inline pkgs/applications/editors/tecoc/default.nix:44]:
```suggestion
  postInstall = ''
```

**Comment by AndersonTorres** [comment]
@GrahamcOfBorg build

**Comment by AndersonTorres** [comment]
Why does it complain about tangranm??

https://gist.github.com/GrahamcOfBorg/2f7d2ee6a76622b59c34a45dba757dd0

**Comment by SuperSandro2000** [comment]
> Why does it complain about tangranm??
> 
> [gist.github.com/GrahamcOfBorg/2f7d2ee6a76622b59c34a45dba757dd0](https://gist.github.com/GrahamcOfBorg/2f7d2ee6a76622b59c34a45dba757dd0)

An old PR got merged. Fixing it right now.

**Comment by SuperSandro2000** [comment]
@ofborg eval

File: pkgs/top-level/lua-packages.nix [hunk 2]
```diff
@@ -5,13 +5,29 @@
    for each package in a separate file: the call to the function would
    be almost as must code as the function itself. */
 
-{ fetchurl, stdenv, lua, unzip, pkg-config
```
Comment (SuperSandro2000) [inline pkgs/top-level/lua-packages.nix:8]: I would prefer it if you would revert this change here.

File: pkgs/applications/networking/instant-messengers/tangram/default.nix [hunk 3]
```diff
@@ -15,7 +31,14 @@ stdenv.mkDerivation rec {
     fetchSubmodules = true;
   };
 
-  buildInputs = [ gdk-pixbuf gjs glib gsettings-desktop-schemas gtk3 webkitgtk ];
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/instant-messengers/tangram/default.nix:18]: I don't think anything got changed here. It would be probably better to revert this.
